client/telemetry/PosthogClient.go

```diff
@@ -38,8 +38,8 @@ type PosthogClient struct {
 type PosthogConfig struct {
 	PosthogApiKey           string `env:"POSTHOG_API_KEY" envDefault:""`
 	PosthogEndpoint         string `env:"POSTHOG_ENDPOINT" envDefault:"https://app.posthog.com"`
-	SummaryInterval         int    `env:"SUMMARY_INTERVAL" envDefault:"24"`
-	HeartbeatInterval       int    `env:"HEARTBEAT_INTERVAL" envDefault:"3"`
+	SummaryCronExpr         string `env:"SUMMARY_CRON_EXPR" envDefault:"0 0 * * *"`     // Run once a day, midnight
+	HeartbeatCronExpr       string `env:"HEARTBEAT_CRON_EXPR" envDefault:"0/5 * * * *"` // Run every 5 min
```

Comment:
I think this is very high frequency... make it once every 6 hours... in fact we can use summary also as heartbeat

Reply:
we can configure it in Environment Variable